### DISCUSSION TITLE
2D Ortho-Camera not panning correctly

### DISCUSSION THREAD

**Discussion by DavidMedin** [bevyengine/bevy#3243] [Q&A] [answered]
I am trying to render an image on the screen and pan the camera with dragging the mouse on left click.
My camera entity is made like this in the setup function ...
```commands.spawn_bundle(OrthographicCameraBundle::new_2d());```
Later in the setup function I made a image like such ...
```
let transform = Transform {
    translation: Vec3::new(0.0,0.0,0.0),
    scale: Vec3::splat(6.0),
    ..Default::default()
};
let sprite = TextureAtlasSprite {
    index: match room_i {
       /* code to find index */
    },
    ..Default::default()
};
commands.spawn_bundle(SpriteSheetBundle {
    texture_atlas: texture_atlas_handle.clone(),
    transform: transform,
    sprite : sprite,
    ..Default::default()
});
```
I also made a system to pan the camera. This is the only system that runs ever frame ...
```
fn mouse_moved(
    mut mouse_motion_events : EventReader<MouseMotion>
    ,mut cameras: Query<(&Camera,&mut Transform)>
    ,mouse_button_input: Res<Input<MouseButton>>
){
    if mouse_button_input.pressed(MouseButton::Left){
        for event in mouse_motion_events.iter(){
            for (camera,mut pos) in cameras.iter_mut() {
                pos.translation += Vec3::new(-event.delta.x,event.delta.y,0.0);
            }
        }
    }
}
```

This mostly works, however the camera moves faster than the camera, therefor, is not in sync with the cursor.
At one point I suspected it was something to do with camera scale, but I checked all the scale members of the components of the camera and found the scales were all fine (1 or identity).
What could be causing the panning of the camera to not be 1:1 with cursor movement?

**Comment by DJMcNab** [comment, marked as answer]
Mouse motion doesn't take into account cursor acceleration

You probably want `CursorMoved`
